test/verify/check-services

```diff
@@ -267,6 +267,15 @@ Unit=test.service
         self.toggle_onoff()
         self.check_service_details(["Running", "Automatically starts"], ["Reload", "Restart", "Stop", "Disallow running (mask)"], True)
 
+        # Survives a burst of events
+        b.go("#/")
+        self.wait_service_present("test.service")
+        m.execute("udevadm trigger; udevadm settle")
+        b.click(self.svc_sel("test.service"))
+        self.check_service_details(["Running", "Automatically starts"], ["Reload", "Restart", "Stop", "Disallow running (mask)"], True)
+        self.toggle_onoff()
```

Comment:
This disables test.service, but the rest of the test expects it to be enabled.  That's the reason why it has failed everywhere, not the additional stress from `udevadm trigger`.

Reply:
Argh, indeed -- #13958 has the correct version.